spec/controllers/account_reset/delete_account_controller_spec.rb

```diff
@@ -8,23 +8,24 @@
   end
   let(:invalid_token_error) { { token: [invalid_token_message] } }
 
+  before { stub_analytics }
   describe '#delete' do
     it 'logs a good token to the analytics' do
-      user = create(:user, :fully_registered, :with_backup_code)
+      user = create(:user, :fully_registered, :with_backup_code, confirmed_at: Time.zone.now.round)
```

Comment:
how come we have the `.round` if we're only checking for `kind_of(Time)` below? I'd expect the `.round` if we checked for specific values

Reply:
nevermind I got the specs confused, I see we're using the time in just this spec